pyincore/analyses/residentialbuildingrecovery/residentialbuildingrecovery.py

```diff
@@ -183,7 +183,7 @@ def household_income_prediction(income_groups, num_samples):
                 # Now reassemble into Pandas DataFrame
                 group['sample_{}'.format(i)] = group_samples[i, :]
 
-            prediction_results = prediction_results.append(group, ignore_index=True)
+            prediction_results = prediction_results.concat(group, ignore_index=True)
```

Comment:
I believe this needs to be:
pd.concat([prediction_results, group], ignore_index=True)

@nunezco2 can you confirm this?